### PR TITLE
Rate conversation mailer

### PR DESCRIPTION
**Pull request by tejaswinichile**
Fixes: https://linear.app/chatwoot/issue/CW-1363/rate-the-conversation-is-sent-as-a-message-in-the-emails

### CODE REVIEW

**Comment by tejaswinichile** [comment]
Screenshot 2023-05-03 at 3 32 04 PM
Screenshot 2023-05-03 at 3 32 12 PM

I have added the link in new mailer and added the link to rate the conversation.

File: app/javascript/dashboard/components/widgets/conversation/Message.vue [hunk 1]
```diff
@@ -250,7 +250,11 @@ export default {
       );
 
       if (this.contentType === 'input_csat') {
-        return this.$t('CONVERSATION.CSAT_REPLY_MESSAGE') + botMessageContent;
+        const link = `/survey/responses/${this.data.conversation_id}`;
```
Comment (tejaswinichile) [inline app/javascript/dashboard/components/widgets/conversation/Message.vue:253]: @muhsin-k / @iamsivin Can you help me to add a link here in this message?

**Comment by tejaswinichile** [comment]
Test Steps:

1. Enable CSAT for inbox.
2. Resolve conversation for inbox.
3. It should include the CSAT survey link in the message so it would include further in the summary email.

File: app/javascript/dashboard/i18n/locale/en/conversation.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "CONVERSATION": {
     "SELECT_A_CONVERSATION": "Please select a conversation from left pane",
-    "CSAT_REPLY_MESSAGE": "Please rate the conversation",
+    "CSAT_REPLY_MESSAGE": "Please rate the conversation: %{link}",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/conversation.json:4]: @tejaswinichile We shouldn't show this to the agents.

Comment (pranavrajs) [reply app/javascript/dashboard/i18n/locale/en/conversation.json:4]: This is only required in the emails sent to the customer.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please see my feedback.

**Comment by tejaswinichile** [comment]
@pranavrajs Please have a look at this:

Screenshot 2023-05-04 at 5 51 51 PM
Screenshot 2023-05-04 at 8 23 21 PM

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**